plugin/web/components/sidebar/tabs/MapSettingTab.tsx

```diff
@@ -0,0 +1,141 @@
+import bgmap_darkmatter from "@web/components/common/Icon/icons/bgmap_darkmatter.svg";
+import bgmap_gsi from "@web/components/common/Icon/icons/bgmap_gsi.svg";
+import bgmap_tokyo from "@web/components/common/Icon/icons/bgmap_tokyo.svg";
+import mapBing from "@web/components/common/Icon/icons/mapBing.svg";
+import { styled } from "@web/theme";
+import { Checkbox, Divider, Radio, Row, Slider, Space, Typography, Button } from "antd";
+import React, { memo } from "react";
+
+const mapViewData = ["3D Terrain", "3D smooth", "2D"];
+const baseMapData = [
+  {
+    key: "1",
+    title: "Aerial photography (Bing)",
+    icon: mapBing,
+  },
+  {
+    key: "2",
+    title: "National latest photo (seamless)",
+    icon: bgmap_tokyo,
+  },
+  {
+    key: "3",
+    title: "GSI Maps (light color)",
+    icon: bgmap_gsi,
+  },
+  {
+    key: "4",
+    title: "Dark Matter",
+    icon: bgmap_darkmatter,
+  },
+];
```

Comment:
These will need i18n support in the future, so please move these in to the component.

Reply:
f0981a019a1257559ab6b090ae8069822b8ac5a3